### ISSUE TITLE
Train Loss seems stuck after a point

### ISSUE THREAD

**Issue by ansh-wiai** [rajpurkarlab/CheXzero#7]
Hi, Thanks for the code you've provided. 

I was trying to run the train on default specs of the repository. However I observed that the training loss gets suck at a value after a few minibatch cycles for both pretrained and non-pretrained cases. I tried this on the complete MIMIC dataset as well as a smaller batch I created for testing, but the loss was getting stuck at a value in all cases rather than moving towards overfitting.

Is this expected behaviour? Has anyone else also come across this?
